src/styles/Footer.css

```diff
@@ -0,0 +1,3 @@
+footer {
+  height: 100%;
```

Comment:
@bill-feng Could you explain why footer's height needs to be 100%?

Reply:
this is a mistake. fixed!